arwen/common.go

```diff
@@ -122,6 +127,10 @@ func (ac *AsyncGeneratedCall) GetGasLimit() uint64 {
 	return ac.GasLimit
 }
 
+func (ac *AsyncGeneratedCall) GetGasLocked() uint64 {
+	panic("not implemented")
```

Comment:
Perhaps return max(uint64)?

Reply:
Not production code - this method falls under the Promises refactor. It will be changed anyway. Currently exists to shut the compiler up until refactoring.

Reply:
still, no panic in production code as we want to run in process arwen.

Reply:
Replaced with `return 0`.